capstone-frontend/src/components/CreateList.jsx

```diff
@@ -28,6 +28,19 @@ class CreateList extends Component {
     }
   }
 
+  initialSelectedBook = () => {
+    if (this.props.selectedBookID && this.props.selectedBook &&
+      !this.state.addedBooksIDs.includes(this.props.selectedBookID) && !this.state.addedBooks.includes(this.props.selectedBook)) {
```

Comment:
Break this up into multiple lines for each condition

Reply:
Ok